addons/sys_external/fnc_initRadio.sqf

```diff
@@ -19,3 +19,6 @@ params ["_radioId"];
 
 [_radioId, false] call FUNC(allowExternalUse);
 [_radioId, "setState", ["isUsedExternally", [false, objNull]]] call EFUNC(sys_data,dataEvent);
+
+// To prevent the GUI from being simultaneously opened.
+[_radioId, "setState", ["isGuiOpened", false]] call EFUNC(sys_data,dataEvent);
```

Comment:
Although I personally don't like states beginning with a verb, this is already the case with other states. So we can leave it as it is